cargo-screeps/README.md

```diff
@@ -13,25 +13,38 @@ This is not fully tested, but feel free to use! Issues are welcome.
 
 ---
 
-`cargo-screeps` has three build options:
+`cargo-screeps` build options:
 
-### `--build`:
+### `build`:
 
 1. during build stage, shell out to https://github.com/koute/cargo-web for actual building the rust code.
 2. strip off header / surrounding function `cargo-web` generates for a generic JS file, since we know we're deploying to node
 3. append call to `__initialize` function which cargo-web generates, using `require('compiled')` to get the WASM bytes
 4. create `target/main.js` containing processed JS and copy WASM file to `target/compiled.wasm`
 
-### `--upload`:
+### `upload`:
 
 1. run build.
 2. read `target/*.js` and `target/*.wasm`, keeping track of filenames
 3. read `screeps.toml` for upload options
 4. upload all read files to server, using filenames as the filenames on the server.
 
-### `--check`:
+### `copy`:
 
-1. run `cargo web --check --target=wasm32-unknown-unknown`
+1. run build.
+2. copy `target/main.js` and `target/compiled.wasm` to `<destination directory>/<branch name>/`
```

Comment:
You are assuming that you are keeping the default names. Was that intentional?

Reply:
I'll add a note mentioning that! It was mostly lazyness.